package.json

```diff
@@ -33,7 +33,7 @@
     "xmldom": "^0.6.0"
   },
   "devDependencies": {
-    "@financial-times/n-gage": "^8.2.0",
```

Comment:
[question, non blocking] is there a reason you're doing away with the caret? I feel like FT brewed packages can be trusted enough to follow semantic versioning